pygsti/tools/matrixtools.py

```diff
@@ -125,47 +94,7 @@ def is_valid_density_mx(mx, tol=1e-9):
     bool
         True if mx is a valid density matrix, otherwise False.
     """
-    return is_hermitian(mx, tol) and is_pos_def(mx, tol) and abs(trace(mx) - 1.0) < tol
```

Comment:
I'm not sure why GitHub thinks I removed this line. If you scroll farther down you'll see a "new" line of code that's identical to this old line of code.